web/compatibility.js

```diff
@@ -542,12 +542,6 @@ if (typeof PDFJS === 'undefined') {
     window.setTimeout(callback, 20);
   }
 
-  var isIOS = /(iPad|iPhone|iPod)/g.test(navigator.userAgent);
```

Comment:
The support comment above this function should probably also be adjusted to remove the mention of iOS.